.github/workflows/ruby.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
```

Comment:
Is this comment right? I think we can bin it, action looks pretty standard to me.

Reply:
Yes, this was just pregenerated, we can bin this 👍